### PR TITLE
Better settings for unit tests

### PR DESCRIPTION
**Pull request by ethantkoenig**
Update default values of `setting.RepoRootPath`, `setting.AppDataPath` to be subdirectories of `/tmp/`.

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

File: models/setup_for_test.go [hunk 1]
```diff
@@ -30,8 +30,8 @@ func TestMain(m *testing.M) {
 	setting.RunUser = "runuser"
 	setting.SSH.Port = 3000
 	setting.SSH.Domain = "try.gitea.io"
-	setting.RepoRootPath = "/repos"
-	setting.AppDataPath = "/appdata"
+	setting.RepoRootPath = "/tmp/repos"
```
Comment (andreynering) [inline models/setup_for_test.go:33]: This certainty won't work on Windows. We should use [os.TempDir](https://golang.org/pkg/os/#TempDir) instead.

```go
filepath.Join(os.TempDir(), "repos")
```

Comment (lunny) [reply models/setup_for_test.go:33]: @ethantkoenig could you send another PR to fix this so that we don't need to revert this one.

**Pull request merged**